### ISSUE TITLE
wavelet ANN not working

### ISSUE THREAD

**Issue by Tsardoz** [helme/ecg_ptbxl_benchmarking#12]
keras and tensorflow are needed for wavelets but are not included in the environment. I tried downloading latest versions but program just hung at this point.

**Comment by helme** [comment]
Hi @Tsardoz 

thanks for pointing out the missing dependencies. I updated the environment to also include keras and tensorflow, please pull and update your environment. 

NOTE: since I used the new version of keras, monitoring validation custom metrics (macro AUC in our case) changed in the new version (this also includes loading models). I was not able in to test this appropriately, but instead changed to monitoring the loss (binary crossentropy in our case). Although this is not optimal, this should at least run.
